test/optim/test_optimize.py

```diff
@@ -1934,6 +1934,29 @@ def test_optimize_acqf_mixed_empty_ff(self):
                 raw_samples=10,
             )
 
+    def test_optimize_acqf_mixed_inter_point_constraints(self):
+        mock_acq_function = MockAcquisitionFunction()
+        with self.assertRaisesRegex(
+            UnsupportedError,
+            expected_regex="Inter-point constraints are not supported for sequential optimization. "
```

Comment:
@AVHopp this line is a bit too long (failed the lint), mind updating this? You can run the pre-commit check locally to make sure lint is ok as described here: https://github.com/pytorch/botorch/blob/main/CONTRIBUTING.md#pre-commit-hooks